components/crashtest/src/tests.rs

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// The tests are in a separate file just to ensure that the
+// main `lib.rs` contains nothing but the public interfance.
```

Comment:
nit: public "interface"

Reply:
My favourite mis-spelling of this is "interfarce" but this one's quite fun too; thanks!